src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+	<div>
+		<m-f-i-hero />
+
+		<mfi-loans-wrapper
```

Comment:
If we don't already have a ticket for it, I recommend adding a ticket to change the `MfiLoansWrapper` to navigate to `/lend/filter` instead of legacy. We've been updating the app to navigate to the new lend page recently, so we don't have to introduce new links to the legacy filter page where we can. It looks like it's a one-line fix in that component, so you could do it with this PR or add a ticket.

Reply:
Nice suggestion, Ill do it whit this PR, lemme update it